### PR TITLE
Add Linea L2

### PR DESCRIPTION
**Pull request by jmederosalvarado**
## Changes

- Adds chainspec and configs for [Linea L2](https://docs.linea.build/)

#### What types of changes does your code introduce?

- [ ] Bugfix (a non-breaking change that fixes an issue)
- [x] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a change that causes existing functionality not to work as expected)
- [ ] Optimization
- [ ] Refactoring
- [ ] Documentation update
- [ ] Build-related changes
- [ ] Other: _Description_

#### Requires testing

- [ ] Yes
- [x] No

#### If yes, did you write tests?

- [ ] Yes
- [ ] No

#### Notes on testing

Need to sync a few times to make sure everything works as expected

#### Requires documentation update

- [x] Yes
- [ ] No

_If yes, link the PR to the docs update or the issue with the details labeled `docs`. Remove if not applicable._

#### Requires explanation in Release Notes

- [x] Yes
- [ ] No

_If yes, fill in the details here. Remove if not applicable._

## Remarks

_Optional. Remove if not applicable._

### CODE REVIEW

File: src/Nethermind/Nethermind.Runner/configs/linea-mainnet_archive.cfg [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "Init": {
+    "ChainSpecPath": "chainspec/linea-mainnet.json",
+    "GenesisHash": "0xb6762a65689107b2326364aefc18f94cda413209fab35c00d4af51eaa20ffbc6",
+    "BaseDbPath": "nethermind_db/linea-mainnet_archive",
+    "LogFileName": "linea-mainnet_archive.logs.txt"
+  },
+  "Merge": {
+    "Enabled": false
+  },
```
Comment (benaadams) [inline src/Nethermind/Nethermind.Runner/configs/linea-mainnet_archive.cfg:10]: Archive also needs pruning off?
```suggestion
  },
  "Pruning": {
    "Mode": "None"
  },
```

**Review by benaadams — APPROVED** [review]

**Review by LukaszRozmej — APPROVED** [review]

File: src/Nethermind/Nethermind.Runner/configs/linea-sepolia_archive.cfg [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "GenesisHash": "0x65a64c825d7c13ce1bf077801d0b6b2a89853e19e4a89a5433d7d85d2102a20b",
+    "BaseDbPath": "nethermind_db/linea-sepolia",
+    "LogFileName": "linea-sepolia.logs.txt"
+  },
+  "Merge": {
+    "Enabled": false
+  }
+  "Pruning": {
+    "Mode": "None"
+  },,
```
Comment (kamilchodola) [inline src/Nethermind/Nethermind.Runner/configs/linea-sepolia_archive.cfg:13]: This is a typo

Comment (kamilchodola) [reply src/Nethermind/Nethermind.Runner/configs/linea-sepolia_archive.cfg:13]: @jmederosalvarado cc

**Comment by kamilchodola** [comment]
General comment: 
Shouldn't each new config have default name for Metrics?
Like: 
"Metrics": {
    "NodeName": "Energy_Web"
  },
  
  ?

**Pull request merged**